scarb-metadata/src/command/metadata_command.rs

```diff
@@ -15,10 +15,7 @@ use crate::{Metadata, VersionPin};
 pub enum MetadataCommandError {
     /// `scarb metadata` command did not produce any metadata
     #[error("`scarb metadata` command did not produce any metadata")]
-    NotFound {
-        /// Captured standard output if any.
-        stdout: String,
-    },
+    NotFound,
```

Comment:
Can we keep that? Shouldn't be too hard I think (the last resort would be to return it from `exec`).